samples/subsys/ipc/openamp/boards/stm32h747i_disco_m7.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2021 BrainCo Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		/*
+		 * shared memory reserved for the inter-processor communication
+		 */
```

Comment:
a reason to use multi-line comment format? 
could be on one line:
`/* shared memory reserved for the inter-processor communication */`

Reply:
No other special reason, change to one line.